Entity/Task.php

```diff
@@ -163,15 +163,15 @@ public function setEntityId(string $entityId): self
         return $this;
     }
 
-    public function getTaskId(): string
+    public function getTaskId(): ?string
```

Comment:
@wachterjohannes can you confirm this?

Reply:
Okay seem like onDelete can make this null: https://github.com/sulu/SuluAutomationBundle/blob/2cfc3a7122a96939d2f203f6fe54bcb74e2cfd46/Resources/config/doctrine/Task.orm.xml#L25

Reply:
yes - it should be nullable - if the ta_task table was cleared (in favor of memory)